apps/dashboard/app/(app)/settings/root-keys/new/client.tsx

```diff
@@ -193,7 +193,7 @@ export const Client: React.FC<Props> = ({ apis }) => {
             {showKeyInSnippet ? snippet : snippet.replace(key.data?.key ?? "", maskedKey)}
             <div className="relative -top-8 right-[88px] flex items-start justify-between gap-4">
```

Comment:
You can use `items-center` instead with `items-start`. And then remove the `mt-1.5` from the `CopyButton`